src/System.Net.Security/ref/System.Net.Security.Manual.cs

```diff
@@ -1,19 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-// ------------------------------------------------------------------------------
-// Changes to this file must follow the http://aka.ms/api-review process.
-// ------------------------------------------------------------------------------
-
-
-namespace System.Security.Authentication.ExtendedProtection
-{
-    public partial class ServiceNameCollection : System.Collections.ICollection
-    {
-        public int Count { get { throw null; } }
```

Comment:
Just to make sure, I suppose all these members are comming from the base class right?